src/navbar/index.js

```diff
@@ -43,29 +43,27 @@ class Navbar extends React.Component {
   }
 
   renderRightContent = () => {
-    const { theme, children } = this.props;
-    const { showMenu } = this.state;
+    const { theme } = this.props;
 
     return (
       <div className={theme.rightContent}>
         {this.renderDesktopView()}
-        {showMenu && this.renderMobileView()}
-        {children && (
-          <span
-            tabIndex={0}
-            className="icon-menu"
-            onClick={this.toggleMenu}
-            onBlur={this.hideMenu}
-          />)
-        }
+        {this.renderMobileView()}
+        <span
+          tabIndex={0}
+          className="icon-menu"
+          onClick={this.toggleMenu}
+          onBlur={this.hideMenu}
+        />
```

Comment:
Can we make this menu Icon to render only when `children` (nav links) are present?

```jsx
{children && (
   <span
     tabIndex={0}
     className="icon-menu"
     onClick={this.toggleMenu}
     onBlur={this.hideMenu}
   />)
}
```
Check the screen shot below.
![screenshot from 2018-12-11 12-46-51](https://user-images.githubusercontent.com/22497932/49784316-e23b3500-fd42-11e8-9cde-cc7c8657067a.png)

Reply:
Done. https://github.com/Codebrahma/react-lite-ui/pull/201/commits/5f7f1fc6267efea9d7bb0c5769dff3f5c7e0335c